apps/files/src/views/Sidebar.vue

```diff
@@ -202,7 +203,10 @@ export default {
 					'star-loading': this.starLoading,
 					active: this.activeTab,
 					background: this.background,
-					class: { 'has-preview': this.fileInfo.hasPreview },
+					class: {
+						'has-preview': this.fileInfo.hasPreview,
```

Comment:
Since we went for proper BEM syntax, maybe also switch to this one?

```suggestion
						'app-sidebar--has-preview': this.fileInfo.hasPreview,
```